### PR TITLE
Update Use Cases in Developer Guide for List

### LINKED ISSUE #301: [PE-D][Tester A] DG Missing Use Case
Might want to consider including a use case for `list`.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Fluffykan/ped#13

### PR DESCRIPTION
**Pull request by colinhia**
Closes #301

### CODE REVIEW

**Review by FionaQY — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -378,13 +378,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-**Use case: U1 - Delete a client**
+**Use case: U1 - List all clients**
 
 **MSS**
 
-1.  User requests to list clients.
-2.  AgentAssist shows a list of clients.
-3.  User requests to delete a specific client in the list.
+1. User requests to see the entire list of clients.
+2. AgentAssist shows a list of clients.
```
Comment (ZShunRen) [inline docs/DeveloperGuide.md:386]:
```suggestion
2. AgentAssist shows a list of all clients.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -478,11 +487,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
     Use case resumes at step 3.
 
-**Use case: U5 - View a client**
+**Use case: U6 - View a client**
 
 **MSS**
 
-1.  User performs <u>Filter for a client (U3)</u> or requests to list all clients.
+1.  User performs <u>Filter for a client (U4)</u> or requests to <u>List all clients (U1)</u>.
 2.  AgentAssist returns a list of people, with the client in it.
```
Comment (ZShunRen) [inline docs/DeveloperGuide.md:495]:
```suggestion
2.  AgentAssist returns a list of clients, with the client of interest in it.
```

**Review by ZShunRen — APPROVED** [review]
LGTM, just a few suggestions.

**Pull request merged**